web/themes/custom/westy/templates/content/node--page--nested.html.twig

```diff
@@ -1,5 +1,7 @@
 {% if content.field_parent.0|length > 0 %}
```

Comment:
Can we remove these top 3 lines? It looks like that `parent_path` variable isn't used anymore.